src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/QinQFlowSpec.groovy

```diff
@@ -842,7 +842,9 @@ class QinQFlowSpec extends HealthCheckSpecification {
         assumeTrue("Unable to find required switches in topology", (allTraffGenSwitches.size() > 1))
         def swP = topologyHelper.getAllNeighboringSwitchPairs().find {
             [it.src, it.dst].every { sw ->
-                sw.dpId in allTraffGenSwitches*.dpId && northbound.getSwitchProperties(sw.dpId).multiTable
+                sw.dpId in allTraffGenSwitches*.dpId && northbound.getSwitchProperties(sw.dpId).multiTable &&
+                        northbound.getSwitchProperties(sw.dpId).supportedTransitEncapsulation
+                                .contains(FlowEncapsulationType.VXLAN.toString().toLowerCase())
```

Comment:
not sure why it's required here. I don't see any vxlan context in this test

Reply:
looks like my mistake. fixed